requirement/main.txt

```diff
@@ -17,3 +17,4 @@ retrying
 Dickens==1.0.1
 signalled-timeout==1.0.0
 smart_open==1.5.3
+s3fs
```

Comment:
I didn't know about this library, very cool! From the dask organization - I am wondering if it would be advantageous for us to support Dask dataframes for lower-memory environments, at least for non-training operations like matrix building and prediction. It's probably not a good fit but it's a possibility.